benchmarks/WebJobs.Script.Benchmarks/RecommendedConfig.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using BenchmarkDotNet.Columns;
+using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Diagnosers;
+using BenchmarkDotNet.Exporters;
+using BenchmarkDotNet.Exporters.Json;
+using BenchmarkDotNet.Jobs;
+using BenchmarkDotNet.Loggers;
+using BenchmarkDotNet.Reports;
+using Perfolizer.Horology;
+using System.IO;
+
+namespace Microsoft.Azure.WebJobs.Script.Benchmarks
+{
+    /// <summary>
+    /// Default config largely matching the dotnet/performance repo:
+    /// https://github.com/dotnet/performance/blob/a453235176943a222f126762b4b5436f6687d62e/src/harness/BenchmarkDotNet.Extensions/RecommendedConfig.cs
+    /// </summary>
+    internal static class RecommendedConfig
+    {
+        public static IConfig Create(DirectoryInfo artifactsPath)
+        {
+            var job = Job.Default
+                .WithWarmupCount(1) // 1 warmup is enough for our purpose
+                .WithIterationTime(TimeInterval.FromMilliseconds(250)) // the default is 0.5s per iteration, which is slighlty too much for us
+                .WithMinIterationCount(15)
+                .WithMaxIterationCount(20); // we don't want to run more that 20 iterations
+
+            return DefaultConfig.Instance
+                .AddLogger(ConsoleLogger.Default) // log output to console
+                .AddExporter(MarkdownExporter.GitHub) // export to GitHub markdown
```

Comment:
Pretty cool that it has a GitHub exporter, I wonder if we would want to utilise this in a GitHub Action so we can do something like `/benchmark` or is the plan to eventually run these in CI at all?

Reply:
I think plugging these in would be a great next step and absolutely something I'd like to do - but I'm not sure the best (most usable) way. Current intent is to reach out to some people out for the year (e.g. Adam Sitnik) to see what other teams are doing here and advice on what we can best utilize. If you or anyone else has ideas here, maybe we should start an issue on that integration? Was pacing myself on not creating _too_ much noise while people are out :)

Reply:
Created an issue! #7998